internal/tiltfile2/tiltfile_test.go

```diff
@@ -703,6 +726,42 @@ func (f *fixture) assertManifest(name string, opts ...interface{}) model.Manifes
 			if opt.num != len(f.entities(m)) {
 				f.t.Fatalf("manifest %v has %v entities in %v; expected %v", m.Name, len(f.entities(m)), m.K8sYAML(), opt.num)
 			}
+
+		case matchPathHelper:
+			// Make sure the path matches one of the mounts.
+			isDep := false
+			path := f.JoinPath(opt.path)
+			for _, d := range m.LocalPaths() {
+				_, isChild := ospath.Child(d, path)
+				if isChild {
+					isDep = true
+				}
+			}
+
+			if !isDep {
+				f.t.Errorf("Path %s is not a dependency of manifest %s", path, m.Name)
+			}
+
+			expectedFilter := opt.missing
+			filter := ignore.CreateBuildContextFilter(m)
+			filterName := "BuildContextFilter"
+			if opt.fileChange {
+				var err error
+				filter, err = ignore.CreateFileChangeFilter(m)
+				assert.NoError(f.t, err)
+				filterName = "FileChangeFilter"
+			}
+
+			actualFilter, err := filter.Matches(path, false)
+			assert.NoError(f.t, err)
```

Comment:
this won't stop exec if there IS an error, so either nest the stuff below in
```if assert.NoError(...) {
    ...
}```
or `f.t.Error` if there IS an error.

Reply:
i changed it to a Fatal. if either if these fail, we probably want to stop everything, since they're really not supposed to fail.